src/main/java/org/broadinstitute/dsde/consent/ontology/model/DataUseV3.java

```diff
@@ -218,63 +217,63 @@ public String toString() {
         sb.append(DataUseV3.class.getName()).append('@').append(Integer.toHexString(System.identityHashCode(this))).append('[');
         sb.append("generalUse");
         sb.append('=');
-        sb.append(((this.generalUse == null)?"<null>":this.generalUse));
+        sb.append((Objects.isNull(this.generalUse)?"<null>":this.generalUse));
```

Comment:
Just a generic note here, not a request for a change. This is better style from a dev perspective. This code was auto-generated, so if we ever re-generate it, these changes would be overwritten.